x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/display_settings/constants.ts

```diff
@@ -10,7 +10,7 @@ import { i18n } from '@kbn/i18n';
 export const LEAVE_UNASSIGNED_FIELD = i18n.translate(
```

Comment:
I would do `export const LEAVE_UNASSIGNED_FIELD = '';` here. Also, the translations will need to be removed from [here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/translations/translations/ja-JP.json#L8066) and [here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/translations/translations/zh-CN.json#L8134) when you delete the translation

Just remove these lines, respectively:
```JavaScript
"xpack.enterpriseSearch.workplaceSearch.contentSources.displaySettings.leaveUnassigned.field": "割り当てなし",

```

```JavaScript
"xpack.enterpriseSearch.workplaceSearch.contentSources.displaySettings.leaveUnassigned.field": "不分配",
```